tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -503,12 +503,12 @@ Object.Base {
 
 	route."10" {
 		source	"copier.module.8.2"
-		sink	"copier.module.7.2"
```

Comment:
@RanderWang IIRC we added the second copier because the gain had to be connected to a module within the same pipeline. Why are we removing this now? Because we're not testing the ref FW anymore?

Reply:
@ranj063 . yes we got a conclusion that we don't support any ref FW restriction now. please check Liam's comments : https://github.com/thesofproject/sof/pull/6783#issuecomment-1370912752